### PR TITLE
fix(editor): Added autofocus to input in typeahead dropdowns.

### PR DESCRIPTION
**Pull request by eozan**
SDPA-104

### CODE REVIEW

**Comment by akintolga** [comment]
Hey @eozan check this PR: https://github.com/superdesk/superdesk/pull/1661, there are few other with the typeahead.

File: client/app/scripts/superdesk/directives/sdTypeahead.js [hunk 1]
```diff
@@ -147,9 +147,15 @@ define(['angular'], function(angular) {
                         if (visible || scope.alwaysVisible) {
                             $list.show();
                             scope.hide = false;
+                            $timeout(function() {
+                                $input.focus();
+                            });
```
Comment (petrjasek) [inline client/app/scripts/superdesk/directives/sdTypeahead.js]: would be good to add `0, false` to `$timeout`, there is no need to trigger digest cycle for this

**Comment by eozan** [comment]
@akintolga I don't think there will be any conflict, but I will keep an eye on it, thanks.

**Comment by akintolga** [comment]
:+1:

**Comment by vied12** [comment]
:ok_hand:

**Pull request merged**